llama_stack/distribution/routers/routers.py

```diff
@@ -154,12 +154,12 @@ async def register_shield(self, shield: ShieldDef) -> None:
 
     async def run_shield(
         self,
-        identifier: str,
+        shield_type: str,
```

Comment:
we actually want this to be identifier. we should just change the meta reference signature to have identifier.